### DISCUSSION TITLE
Fan undervolt speeds

### DISCUSSION THREAD

**Discussion by NatasKrad** [Rem0o/FanControl.Releases#854] [Q&A]
Is it safe (wont damage fans) if I configure a speed curve that drops below the point fan stops spinning?
Thanks

**Comment by Rem0o** [comment]
Yes and no.  That's what the start % and stop % are for. You don't want to run it at a > 0 % where it doesn't spin. If it doesn't spin, you want to be at 0,

**Comment by NatasKrad** [comment]
Thanks. Adjustments made.

Are there any examples/tutorials that show Fan Control configured for 0% to X temp then jumping to min fan start speed and up?
